### PR TITLE
New package: KuroGames.WutheringWaves version 1.5.1.0

### PR DESCRIPTION
**Pull request by solomoncyj**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154715)

### CODE REVIEW

**Comment by Trenly** [comment]
This application doesn't appear to write any ARP entries, or didn't install when I tried it
```
Found KRInstall Wuthering Waves Overseas [KuroGames.WutheringWaves] Version 1.5.1.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://mirrors-package-mc.aki-game.net/client/download/pclauncher_ucValwzcms5hdgbHaflgz6qm/WutheringWaves-overseas-setup-1.5.1.0.exe
  ██████████████████████████████  84.2 MB / 84.2 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs>
```

File: manifests/k/KuroGames/WutheringWaves/1.5.1.0/KuroGames.WutheringWaves.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.WutheringWaves
+PackageVersion: 1.5.1.0
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司
+PackageName: KRInstall Wuthering Waves Overseas
```
Comment (Trenly) [inline manifests/k/KuroGames/WutheringWaves/1.5.1.0/KuroGames.WutheringWaves.locale.en-US.yaml:8]:
```suggestion
PackageName: Wuthering Waves
```
This should match Control Panel

File: manifests/k/KuroGames/WutheringWaves/1.5.1.0/KuroGames.WutheringWaves.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.WutheringWaves
+PackageVersion: 1.5.1.0
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司
```
Comment (Trenly) [inline manifests/k/KuroGames/WutheringWaves/1.5.1.0/KuroGames.WutheringWaves.locale.en-US.yaml:7]:
```suggestion
Publisher: KURO TECHNOLOGY (HONG KONG) CO., LTD.
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
Downloaded and ran the installer manually - seems the application does write ARP entries. This means something in the manifest is incorrect - either the installer type or the silent switches, perhaps a missing dependency, not sure what it is

**Comment by solomoncyj** [comment]
.

**Comment by stephengillie** [comment]
This installer didn't appear to install anything. 
- The installer doesn't respond to `/?` nor `--help`. 
- The developer's [download and install guide](https://wutheringwaves.gg/download/) doesn't mention silent installer switches. 
- Running the installer interactively requires agreements:

image

**Pull request closed without merging**